packages/zowe-explorer-api/src/profiles/ProfilesCache.ts

```diff
@@ -59,7 +59,7 @@ export class ProfilesCache {
     protected defaultProfileByType = new Map<string, imperative.IProfileLoaded>();
     protected profileManagerByType = new Map<string, imperative.CliProfileManager>();
     public constructor(protected log: imperative.Logger, protected cwd?: string) {
-        this.cwd = cwd != null ? fs.realpathSync.native(cwd) : undefined;
+        this.cwd = cwd != null ? fs.realpathSync(cwd) : undefined;
```

Comment:
Curious about this particular change.
I'm afraid that this could cause ZE and CLI to not be able to retrieve credentials from the vauls as the path referencing the vault property will contain a lower-case letter for the drive
`c:/path/to/config`

Reply:
Perhaps we could convert the path string to become all uppercase?

Reply:
That won't work as path are case sensitive on *nix environments 😢